### ISSUE TITLE
feat(action): join with line above

### ISSUE THREAD

**Issue by wongjiahau** [ki-editor/ki-editor#494]
Join with line above (JWLA) is operationally symmetrical to Break, where it joins the current selection with the immediate line above, but it doesn’t join the current potentially multiline selection into a single line.

This operation makes Break reversible and vice versa.

**Comment by jcowgar** [comment]
I think this is the same as #422?

**Comment by rluiten** [comment]
I think this is a good idea.

I'd also add that the current Join requires selection, I think there might be benefit if the current join would join next line to current. I find having to select first for join currently more than seems necessary.

**Comment by wongjiahau** [comment]
Agree, JWLA is more orthogonal than the current Join Into Single Line (JISL).

I'm thinking of putting JISL under Transform (`F` in Qwerty) and JWLA takes the position of `I` (Qwerty), what do you think?

**Comment by rluiten** [comment]
JWLA is the opposite direction of what I am thinking of though.
Just from experience in editors its more common to join next to current.
Possibly you need both - which complicates things as you have a symmetric pair.
